runtime/devnet/src/extensions.rs

```diff
@@ -118,9 +125,9 @@ where
 {
 	let charged_dispatch_weight = env.charge_weight(call.get_dispatch_info().weight)?;
 
-	log::debug!(target:LOG_TARGET, "{} inputted RuntimeCall: {:?}", log_prefix, call);
+	log::debug!(target:LOG_TARGET, "{} Inputted RuntimeCall: {:?}", log_prefix, call);
 
-	origin.add_filter(AllowedApiCalls::contains);
+	origin.add_filter(AllowedPopApiCalls::contains);
```

Comment:
Please remove Pop, we dont need to prefix everything with Pop, it just adds noise.